packages/fxa-settings/src/components/DataBlock/index.stories.tsx

```diff
@@ -13,6 +13,10 @@ export default {
   decorators: [withLocalization],
 } as Meta;
 
+export const NEWSingleCodeInlineCopy = () => (
```

Comment:
Nitpick, but I think the all-caps `NEW` might diverge from our usual conventions of naming vars, and it might be clearer to just have it as `SingleCodeInlineCopy`. I'm open to it though if you want to keep it!